connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -330,7 +331,8 @@ public void testCreateConnector() throws Exception {
         // CONN2 is new, should succeed
         configBackingStore.putConnectorConfig(CONN2, CONN2_CONFIG);
         PowerMock.expectLastCall();
-        ConnectorInfo info = new ConnectorInfo(CONN2, CONN2_CONFIG, Collections.<ConnectorTaskId>emptyList());
+        ConnectorInfo info = new ConnectorInfo(CONN2, CONN2_CONFIG, Collections.<ConnectorTaskId>emptyList(),
+            ConnectorType.SOURCE);
```

Comment:
Is there a test that ever returns `ConnectorType.SINK`? Might be good, just to catch things like accidentally always returning `ConnectorType.SOURCE`. WDYT?

Reply:
CONN2 says sourceB for which ConnectorType.SOURCE is appropriate.

Reply:
Right. I'm suggesting adding or changing tests so that we cover both.

Reply:
SINK type is covered in ConnectorsResourceTest